demisto_sdk/commands/unify/unifier.py

```diff
@@ -220,43 +226,27 @@ def insert_script_to_yml(self, script_type, yml_text, yml_data):
 
         clean_code = self.clean_python_code(script_code)
 
-        lines = ['|-']
-        lines.extend(u'    {}'.format(line) for line in clean_code.split('\n'))
-        script_code = u'\n'.join(lines)
-
-        if self.dir_name == 'Scripts':
-            if yml_data.get('script'):
-                if yml_data['script'] != '-' and yml_data['script'] != '':
-                    raise ValueError("Please change the script to be blank or a dash(-) for package {}"
-                                     .format(self.package_path))
-
-        elif self.dir_name == 'Integrations' or self.dir_name == 'Beta_Integrations':
-            if yml_data.get('script', {}).get('script'):
-                if yml_data['script']['script'] != '-' and yml_data['script']['script'] != '':
-                    raise ValueError("Please change the script to be blank or a dash(-) for package {}"
-                                     .format(self.package_path))
-        else:
-            raise ValueError('Unknown yml type for dir: {}. Expecting: Scripts/Integrations'.format(self.package_path))
+        if self.is_script_package:
+            if yml_data.get('script') not in ('', '-'):
+                raise ValueError(f'Please change the script to be blank or a dash(-) for package {self.package_path}')
```

Comment:
why not just replace the script instead of asking the user to replace it with dash

Reply:
we can switch that to a warning, but i don't think we should modify the original files.

Reply:
we should, this is not normal situation that they have something in the script. It is better UX in this case